Bluetooth: Mesh: Mark as internal function

LingaoM opened the pull request:
Marks funcs:`show_faults` as internal, avoiding conflict.

Signed-off-by: Lingao Meng <menglingao@xiaomi.com>

LingaoM committed:
Bluetooth: Mesh: Mark as internal function

Marks funcs:`show_faults` as internal, avoiding conflict.

Signed-off-by: Lingao Meng <menglingao@xiaomi.com>

trond-snekvik commented on tests/bluetooth/tester/src/mesh.c:
```diff
@@ -233,7 +233,7 @@ BT_MESH_HEALTH_PUB_DEFINE(health_pub, CUR_FAULTS_MAX);
 };
 
-void show_faults(uint8_t test_id, uint16_t cid, uint8_t *faults, size_t fault_count)
+static show_faults(uint8_t test_id, uint16_t cid, uint8_t *faults, size_t fault_count)
```
Need to keep the `void`

LingaoM committed:
Tester: Bluetooth: Mesh: Rename function avoiding conflict

Since `write` or `read` belong with system standard API, so rename
this to avoiding conflict.

Signed-off-by: Lingao Meng <menglingao@xiaomi.com>
